src/controllers/sendResetToken.js

```diff
@@ -0,0 +1,104 @@
+import bcrypt from 'bcrypt';
+import { randomBytes } from 'crypto';
+import user from '../db/models';
+import sendgrid from '../middlewares/helper';
+
+const { User } = user;
+
+/**
+  * @description class representing reset user password
+  *
+  * @class SendResettoken
+  */
+class SendResettoken {
+  /**
+    * @description - This method is responsible for creating creating password reset token
+    *
+    * @static
+    * @param {object} request - Request sent to the router
+    * @param {object} response - Response sent from the controller
+    *
+    * @returns {object} - object representing response message
+    *
+    * @memberof UserHandler
+    */
+  static async resetToken(request, response) {
+    try {
+      const userData = await User.findOne({
+        where: { email: request.body.email }
+      });
+      if (!userData) {
+        return response.status(400).json({
+          status: 'Fail',
+          message: 'An email has been sent to your mail',
+        });
+      }
+      const token = randomBytes(40).toString('hex');
```

Comment:
Hi Emeka, what does this line do?

Reply:
The randomBytes generates a token which is converted to string in hexadecimal format.

Reply:
Why does it need to be converted to hexadecimal?

Reply:
it's more secured sending a random string.